themes/README.rst

```diff
@@ -154,3 +176,56 @@ name in the ``@import`` line.
 Run the ``update_assets`` command to recompile the theme::
 
     $ paver update_assets lms --settings=aws
+
+Microsites
+==========
+
+If you want to continue using the "Microsites" theming system, there are a few
+changes you'll need to make. A few templates have been renamed, or folded into
+other templates:
+
+* ``header_extra.html`` has been renamed to ``head-extra.html``. This file
+  was always inserted into the ``<head>`` element of the page, rather than
+  the header of the ``<body>`` element, so this change makes the name more
+  accurate.
+
+* ``google_analytics.html`` has been removed. The contents of this template
```

Comment:
This file was removed because it was unnecessary. I agree that the convention of having one file per vendor is nice, but that's not what we're seeing here -- microsites was simply including `google_analytics.html`, without regards to the contents of that file, and there was no option to include other files like `segment.html` or `fb_analytics.html`, etc. In fact, if you look at the companion pull request that I have in the edx-microsite repository, you'll see that the `google_analytics.html` file was already being misused -- some microsites were using it for facebook analytics, for example.

There's no reason why templates can't include other templates. If theme authors want to split up their theme into multiple files (which is a great way to improve maintainability), then they can simply import those other files from their `head-extra.html` file. That way, they can name the files based on their contents!